nixos/modules/services/network-filesystems/openafs-server.nix

```diff
@@ -0,0 +1,119 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.openafsServer;
+  openafsPkgs = config.services.openafsServer.openafsPkgs;
+  netInfo = if (cfg.advertisedAddresses != []) then
```

Comment:
The construction of `netInfo` reads poorly for me, can it be restructured somewhat?
It seems it'll produce either an empty string or a derivation, which is irregular.